lib/utils.h

```diff
@@ -25,6 +25,24 @@
 #include <cctype>
 #include <cstddef>
 #include <string>
+#include <vector>
+
+inline void splitString(const std::string &str, std::vector<std::string> &stringList, char delimeter)
```

Comment:
This is not used anywhere. I suggest you remove it.

Reply:
Is removed now